lib/desktop/screens/desktop_profile_basic_info/widgets/desktop_basic_info_widget.dart

```diff
@@ -71,10 +72,18 @@ class DesktopBasicInfoWidget extends StatelessWidget {
           ),
           Expanded(
             child: Container(
-              child: Text(
-                data.value ?? '',
-                style: appTheme.textTheme.bodyText2?.copyWith(
-                  color: appTheme.primaryTextColor,
+              child: GestureDetector(
```

Comment:
Could you change GestureDetector -> InkWell Widget?

Reply:
Its just a text link, that is why I have used GestureDetecter